optaplanner-persistence/pom.xml

```diff
@@ -30,4 +30,27 @@
     <module>optaplanner-persistence-jpa</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>validate</id>
+              <phase>validate</phase>
+              <goals>
+                <goal>check</goal>
+              </goals>
+              <configuration>
+                <suppressionsLocation>checkstyle-suppressions.xml</suppressionsLocation>
```

Comment:
This file isn't important enough to live in the root folder. Let's move it into src/(main/?)/checkstyle or something.

Actually, SupressWarning works on a class level (not a package level unfortunately), which is much closer to the actual code (and seen by codacy and other services that rate our quality, etc while this is not) - I 'd much rather just see a SurpressWarning annotation on all the affected classes (even though those are bunch...). Wdyt?